Sources/GetStartedApp/Controller.swift

```diff
@@ -105,7 +105,7 @@ public class Controller {
   * </code>
   */
   public func addVisitors(user: [String: String], respondWith: @escaping ([String: String]?, RequestError?) -> Void) {
```

Comment:
This function could take and return a User class:
```
public func addVisitors(user: User, respondWith: @escaping (User?, RequestError?) -> Void)
```
This would simplify the code inside.